plugin-packages/multimedia/test/src/audio-component.spec.ts

```
@@ -0,0 +1,260 @@
+import { spec } from '@galacean/effects';
+import { generateGUID, Player } from '@galacean/effects';
+import { AudioComponent } from '@galacean/effects-plugin-multimedia';
+
+export interface AudioCompositionOptions {
+  duration: number,
+  endBehavior: spec.EndBehavior,
+  audios: spec.AssetBaseOptions[],
+  start: number,
+  options: spec.AudioContentOptions,
+}
```

Biome

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)